openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -337,10 +343,17 @@ def test_jwt_access_token_scopes_and_filters(self, grant_type):
             grant_type=grant_type,
         )
 
+    def test_asymmetric_jwt_access_token(self):
+        """
+        Verify the JWT is asymmetric.
```

Comment:
Nit:
```suggestion
        Verify the JWT is asymmetric when requested.
```
Hopefully it is easy for you to just accept these changes from inside github. I know they are very minor nits, which you can also ignore.